prov/util/src/util_mr_map.c

```diff
@@ -108,22 +108,37 @@ int ofi_mr_map_verify(struct ofi_mr_map *map, uintptr_t *io_addr,
 	void *addr;
 
 	node = ofi_rbmap_find(map->rbtree, &key);
-	if (!node)
-		return -FI_EINVAL;
+	if (!node) {
+                FI_WARN(map->prov, FI_LOG_MR,
+                        "verify_addr: unknown key: %" PRIu64 "\n", key);
+	        return -FI_EINVAL;
+        }
 
 	attr = node->data;
 	assert(attr);
 
 	if ((access & attr->access) != access) {
-		FI_DBG(map->prov, FI_LOG_MR, "verify_addr: invalid access\n");
+                FI_WARN(map->prov, FI_LOG_MR,
+                        "verify_addr: invalid access: permitted %s\n",
+                        fi_tostr(&attr->access, FI_TYPE_MR_MODE));
+                FI_WARN(map->prov, FI_LOG_MR,
+                        "verify_addr: invalid access: requested %s\n",
+                        fi_tostr(&access, FI_TYPE_MR_MODE));
 		return -FI_EACCES;
 	}
 
 	addr = (void *) (*io_addr + (uintptr_t) attr->offset);
 
-	if ((addr < attr->mr_iov[0].iov_base) ||
+        if ((addr < attr->mr_iov[0].iov_base) ||
```

Comment:
nit: You indented with spaces instead of a tab here.  Please revert

Reply:
Fixed. Editor issues.